packages/amplify-cli-core/src/feature-flags/featureFlags.ts

```diff
@@ -476,6 +476,14 @@ export class FeatureFlags {
 
   // DEVS: Register feature flags here
   private registerFlags = (): void => {
+    this.registerFlag('advancedCompute', [
+      {
+        name: 'enabled',
```

Comment:
It seems wrong to me to have the name of the flag be "enabled". I would register the flag as
```
this.registerFlag('api', [
  {
    name: 'enableElasticContainerOptions'
    ...
  }
]

Reply:
This is for testing, it will me removed in favor of `amplify configure`